server/endpoints_v2.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"net/http"
+	"path/filepath"
+
+	httputils "github.com/RedHatInsights/insights-operator-utils/http"
+	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	// ReportEndpointV2 https://issues.redhat.com/browse/CCXDEV-5097
```

Comment:
it is not needed right now, right?

Reply:
yeah, these are the two immediately planned next ones, I just wanted something there to keep the structure

Reply:
cool TY